src/master/develop/operations.md

```diff
@@ -1,6 +1,86 @@
 # Tamper-proof operations
+
 ## State management
 
+### Current State
+
+Current state of immudb provides proof that clients can use to verify immudb:
+
+:::: tabs
+
+::: tab Go
+```go
+    	state, err := client.CurrentState(ctx)
+    	if err != nil {
+    		log.Fatal(err)
+    	}
+
+    	fmt.Printf("current state is : %v", state)
```

Comment:
Extra indentation here is not really needed and makes the code example much wider than what's needed.

Reply:
not sure why it changed. Fixed, but anyway example needs to be updated.